packages/remix-dev/compiler/compilerServer.ts

```diff
@@ -44,7 +44,9 @@ const createEsbuildConfig = (
   let isCloudflareRuntime = ["cloudflare-pages", "cloudflare-workers"].includes(
     config.serverBuildTarget ?? ""
   );
-  let isDenoRuntime = config.serverBuildTarget === "deno";
+  let isDenoRuntime = ["deno", "netlify-edge"].includes(
+    config.serverBuildTarget!
```

Comment:
Let's make this consistent with `isCloudflareRuntime` instead

```suggestion
    config.serverBuildTarget ?? ""
```